### ISSUE TITLE
Georeferencer does not allow to ignore whatever positional info in the input and does havoc due to that

### ISSUE THREAD

**Issue by mmokrejs** [qgis/QGIS#51814]
### What is the bug or the crash?

I spent several monhs in overall, time to time trying to georeference an image. It was roughly georeferenced already what was the culprit.

It worked for some and seemed NOT to work for other files (Georeferencerer did not complain and finished cleanly, the calculated GCP points seemed fine) but the resulting layer/image loaded in QGIS in a completely wrong area. No matter if loaded directly after georeferencing or later of from the resulting raster file. I use EPSG:5514 which also has negative both X and Y coordinates in Czechia but it also breaks if I use WGS84 to derive the GCP points.

Workaround:
Zap the eventually added roughly georeferenced posions fromt the input image. Neither QGIS Georeferencer nor [Freehand raster georeferencer v. 0.8.3](https://plugins.qgis.org/plugins/FreehandRasterGeoreferencer/version/0.8.3/) are able to ignore the input georeferencing values (in my case they were quite inexact). QGIS does havoc and on a side not, _Freehand raster georeferencer_ refuses to operate on the input. Both should just give me warning and ask to proceed with the input while IGNORING the original values.

To get around try:
```
# get rid of S-JTSK negative coords
convert inputfile.tif -format bmp - > outputfile.without_coords.bmp

# maybe the following would be just enough?
exiftran -f -o outputfile.flipped.tif inputfile.tif
```

Fix:
QGIS-3.28.2-Firenze should ignore the original, unwanted georef values in the input image.

### Steps to reproduce the issue

See above.

### Versions

3.28.2-Firenze

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

**Comment by agiudiceandrea** [comment]
@mmokrejs could you please specify if and how this issue is different from #51367 or #51699, #51299, #51197, #44994?
A bunch of Georeferencer issues have been recently fixed. The fixes are currently available in QGIS 3.29.0-Master [nightly build](https://download.osgeo.org/qgis/windows/weekly/?C=M&O=D) and they will be available in QGIS 3.28.4.

**Comment by mmokrejs** [comment]
I had a look to some even before posting. The bug https://github.com/qgis/QGIS/issues/51367 is different one.

The issue https://github.com/qgis/QGIS/issues/51699 could be same but in my case the offset was huge, somewhere totally elsewhere on Earth, if at all. I had to use `-flip -flop` to make it easier to Georeferencer to do it job and discarded the georef data which were huge values -X -Y probably related tot he huge offset. I cannot express myself probably correctly but "normaly" Georeferencer uses the positive X and Y values in the window (local, virtual coordinates, seems you speak of pixels) but in my case it picked those huge negative values. That possibly caused the huge offset because the local/virtual/pixel coords are in range of millions but the EPSG values were a few magnitute orders more negative. But somebody needs to confirm the idea.

The issue https://github.com/qgis/QGIS/issues/51299 could be same case I faced, it just depend how latge are the values in you originally georeferenced image (which nobody warned you about).

The issue #51197 is exactly waht I found as well. But I think I had even had at that time an older version of QGIS. Like mentioned in https://github.com/qgis/QGIS/issues/51197#issuecomment-1404914243 I also have the impression that also the broken GCP points were fetched during my repeated experiments. One should drop the `*.points` files from the source directory they are not picked up. QGIS could show along the way which sources of information were picked. Especially in the map and report files.

Again, issue #44994 is yet another instance of what I hit as well. Regarding https://github.com/qgis/QGIS/issues/44994#issuecomment-1410868157 , there should be a warning message raised tot he user of QGIS and second, an automated way to discard the offending georef values directly inside the Georeferencer GUI. The fix https://github.com/qgis/QGIS/pull/51705 is only partion IMO. At least commit description does not say it does automagically discard the offending georef data.

**Comment by agiudiceandrea** [comment]
If you want you can test if the issue does occur using the latest QGIS 3.29.0-Master [nightly build](https://download.osgeo.org/qgis/windows/weekly/?C=M&O=D) or provide a sample raster layer and exact step and GCP points in order replicate the issue.

**Comment by mmokrejs** [comment]
No, with master I still get an offsetted georef result. The error is about 20-30 m, depending on the number of points used (20 to 40).

QGIS version
3.29.0-Master
QGIS code revision
[48a42c7fae](https://github.com/qgis/QGIS/commit/48a42c7fae)
Qt version
5.15.8
Python version
3.9.16
GDAL/OGR version
3.6.2
PROJ version
9.1.1
EPSG Registry database version
v10.076 (2022-08-31)
GEOS version
3.11.1-CAPI-1.17.1
SQLite version
3.40.1
PDAL version
2.5.0
PostgreSQL client version
unknown
SpatiaLite version
5.0.1
QWT version
6.1.5
QScintilla2 version
2.13.3
OS version
Gentoo Linux

Note the input and out coords are both in EPSG:5514 where the source should be in pixels, right?

![git-master-input](https://user-images.githubusercontent.com/3297852/218453491-611644f2-240e-42d6-9219-de37c194efbe.jpg)

![git-master-result](https://user-images.githubusercontent.com/3297852/218453515-de055072-b023-4403-8fe1-2b15e61a90e7.jpg)

**Comment by mmokrejs** [comment]
Yeah, see the input coords were treated as the pixels after rounding according to report.pdf?
![git-master-pdf-report](https://user-images.githubusercontent.com/3297852/218488658-14b1114d-f3cc-4bb5-9b1b-d1dd57078513.jpg)

I do not understand the huge red lines in the schema close to the beginning of the report.pdf. Some are way long and have opposite=conflicting orientation. Are those shifts detected in the image to be made?

[report.pdf](https://github.com/qgis/QGIS/files/10723294/report.pdf)
